db/model/Gdoc/GdocBase.ts

```diff
@@ -741,8 +750,11 @@ export class GdocBase implements OwidGdocBaseInterface {
         }
     }
 
-    async loadLinkedCharts(knex: db.KnexReadonlyTransaction): Promise<void> {
```

Comment:
We should also make sure that the `getPublishedExplorersBySlug` call in here is also cached, it's also happening thousands of times.

Reply:
Yeah, that's a big bottleneck. I've added it in [this PR](https://github.com/owid/owid-grapher/pull/5535). It'd make sense to first merge that one and then revisit whether this PR helps at all.